frontend/src/components/mine/ContactInfo/PartyRelationships/InactiveContact.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Icon, Card, Row, Col, Button } from "antd";
+import * as router from "@/constants/routes";
+import { Redirect } from "react-router";
+
+const propTypes = {
+  partyRelationshipTypeCode: PropTypes.string.isRequired,
+  partyRelationshipTypeLabel: PropTypes.string.isRequired,
+  mine: PropTypes.object.isRequired,
+};
+
+export class InactiveContact extends Component {
+  state = {
+    redirectToProfile: false,
+  };
+
+  render() {
+    if (this.state.redirectToProfile === true) {
```

Comment:
`this.state.redirectToProfile` is a boolean, so the comparison is redundant

Reply:
True. Changed.